projects/1.20/assets/potionsmaster/potionsmaster/lang/zh_cn.json

```
@@ -0,0 +1,239 @@
+  "item.minecraft.lingering_potion.effect.stronghold_sight": "滞留型要塞快速旅行药水",
+  "item.minecraft.lingering_potion.effect.swamp_hut_sight": "滞留型沼泽小屋快速旅行药水",
+  "item.minecraft.lingering_potion.effect.jungle_pyramid_sight": "滞留型丛林神庙快速旅行药水",
+  "item.minecraft.lingering_potion.effect.desert_pyramid_sight": "滞留型沙漠神殿快速旅行药水",
+  "item.minecraft.lingering_potion.effect.igloo_sight": "滞留型雪屋快速旅行药水",
+  "item.minecraft.lingering_potion.effect.mansion_sight": "滞留型林地府邸快速旅行药水",
+  "item.minecraft.lingering_potion.effect.mineshaft_sight": "滞留型废弃矿井快速旅行药水",
+  "item.minecraft.lingering_potion.effect.pillager_outpost_sight": "滞留型掠夺者前哨站快速旅行药水",
+  "effect.potionsmaster.coalpotioneffect": "煤炭视野",
+  "effect.potionsmaster.ironpotioneffect": "铁视野",
+  "effect.potionsmaster.redstonepotioneffect": "红石视野",
+  "effect.potionsmaster.lapispotioneffect": "青金石视野",
+  "effect.potionsmaster.goldpotioneffect": "金视野",
+  "effect.potionsmaster.diamondpotioneffect": "钻石视野",
+  "effect.potionsmaster.emeraldpotioneffect": "绿宝石视野",
+  "effect.potionsmaster.copperpotioneffect": "铜视野",
+  "effect.potionsmaster.tinpotioneffect": "锡视野",
+  "effect.potionsmaster.netheritepotioneffect": "下界合金视野",
+  "effect.potionsmaster.nickelpotioneffect": "镍视野",
+  "effect.potionsmaster.aluminiumpotioneffect": "铝视野",
+  "effect.potionsmaster.vibraniumpotioneffect": "振金视野",
+  "effect.potionsmaster.unobtainiumpotioneffect": "难得素视野",
+  "effect.potionsmaster.allthemodiumpotioneffect": "ATM视野",
+  "effect.potionsmaster.uraniumpotioneffect": "铀视野",
+  "effect.potionsmaster.leadpotioneffect": "铅视野",
+  "effect.potionsmaster.silverpotioneffect": "银视野",
+  "effect.potionsmaster.osmiumpotioneffect": "锇视野",
+  "effect.potionsmaster.zincpotioneffect": "锌视野",
+  "effect.potionsmaster.quartzpotioneffect": "石英视野",
+  "effect.potionsmaster.bismuthpotioneffect": "铋视野",
+  "effect.potionsmaster.crimsonironpotioneffect": "绯红铁视野",
+  "effect.potionsmaster.platinumpotioneffect": "铂视野",
+  "effect.potionsmaster.villagepotioneffect": "村庄快速旅行",
+  "effect.potionsmaster.shipwreckpotioneffect": "沉船快速旅行",
+  "effect.potionsmaster.ocean_ruinpotioneffect": "海底废墟快速旅行",
+  "effect.potionsmaster.buried_treasurepotioneffect": "埋藏的宝藏快速旅行",
+  "effect.potionsmaster.endcitypotioneffect": "末地城快速旅行",
+  "effect.potionsmaster.fortresspotioneffect": "下界要塞快速旅行",
+  "effect.potionsmaster.monumentpotioneffect": "海底神殿快速旅行",
+  "effect.potionsmaster.strongholdpotioneffect": "要塞快速旅行",
+  "effect.potionsmaster.swamp_hutpotioneffect": "沼泽小屋快速旅行",
+  "effect.potionsmaster.jungle_pyramidpotioneffect": "丛林神庙快速旅行",
+  "effect.potionsmaster.desert_pyramidpotioneffect": "沙漠神殿快速旅行",
+  "effect.potionsmaster.igloopotioneffect": "雪屋快速旅行",
+  "effect.potionsmaster.mansionpotioneffect": "林地府邸快速旅行",
+  "effect.potionsmaster.mineshaftpotioneffect": "废弃矿井快速旅行",
+  "effect.potionsmaster.pillager_outpostpotioneffect": "掠夺者前哨站快速旅行",
+  "bezoar.effect": "治愈中毒",
+  "bladder.effect": "治愈挖掘疲劳",
+  "charcoal.effect": "治愈凋零",

```

可能会和“治愈凋灵”搞混，误认为是治愈某种生物

```suggestion
  "charcoal.effect": "治愈凋零效果",
```